features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -73,13 +74,34 @@ const PoolHeader = ({ tokenA, tokenB, priceBreakdown }: PoolHeaderProps) => (
   </Inline>
 )
 
+const SwapFeeElem = ({
```

Comment:
i'd leave out things like `Elem` from component names - i think that since it's a component, it is implied that it is a DOM element. i think `SwapFee` would be fine 👍